libs/core/src/lib/utils/services/focus-trap.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@angular/core';
+import { createFocusTrap, FocusTrap, Options } from 'focus-trap';
+import { uuidv4 } from '../functions/uuidv4-generator';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class FocusTrapService {
+
+    private _focusTrapInstances: Map<any, FocusTrap> = new Map();
+
+    constructor() { }
+
+    createFocusTrap(element: string | HTMLElement | SVGElement | (string | HTMLElement | SVGElement)[],
```

Comment:
Might be useful to add a comment that method returns id of the created focus trap, what do you think?